targets/mate

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]
```

Comment:
This should be `if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then`; bash complains about an unmatched `fi` otherwise.